module/attribute/tests/Domain/Event/Attribute/AttributeCreatedEventTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Attribute\Tests\Domain\Event\Attribute;
+
+use Ergonode\Attribute\Domain\Entity\AttributeId;
+use Ergonode\Attribute\Domain\Event\Attribute\AttributeCreatedEvent;
+use Ergonode\Attribute\Domain\ValueObject\AttributeCode;
+use Ergonode\Core\Domain\ValueObject\TranslatableString;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ */
+class AttributeCreatedEventTest extends TestCase
+{
+    /**
+     * @param $multilingual
+     * @param $editable
+     * @param $deletable
+     * @param $system
+     *
+     * @dataProvider dataProvider
+     */
+    public function testEventCreation($multilingual, $editable, $deletable, $system): void
+    {
+        /** @var AttributeId|MockObject $attributeId */
+        $attributeId = $this->createMock(AttributeId::class);
+        /** @var AttributeCode | MockObject $attributeCode */
+        $attributeCode = $this->createMock(AttributeCode::class);
+        /** @var TranslatableString | MockObject $label */
+        $label = $this->createMock(TranslatableString::class);
+        /** @var TranslatableString | MockObject $hint */
+        $hint = $this->createMock(TranslatableString::class);
+        /** @var TranslatableString | MockObject $placeholder */
+        $placeholder = $this->createMock(TranslatableString::class);
+        $type = 'string';
+        $class = 'class';
+        $parameters = [];
+
+        $event = new AttributeCreatedEvent($attributeId, $attributeCode, $label, $hint, $placeholder, $multilingual, $type, $class, $parameters, $editable, $deletable, $system);
```

Comment:
Is to long, please change to:
` $event = new AttributeCreatedEvent(

$attributeId, 
$attributeCode, 
$label, 
$hint, 
$placeholder,
$multilingual,
$type,
$class,
$parameters,
$editable,
$deletable, 
$system);`